test/integration/anonymous.toml

```diff
@@ -209,8 +208,8 @@ definitions = '''
       "staticSize": 24,
       "exclusiveSize": 10,
       "members": [
-        {"name":"__anon_member_0", "staticSize":2, "exclusiveSize":2},
-        {"name":"__anon_member_1", "staticSize":8, "exclusiveSize":8},
+        {"name":"__oi_anon_0", "staticSize":2, "exclusiveSize":2},
+        {"name":"__oi_anon_2", "staticSize":8, "exclusiveSize":8},
```

Comment:
A bit annoying that this is "_2" not "_1", but it's because of padding members. Didn't think it was worth fixing right now